website/content/docs/k8s/connect/terminating-gateways.mdx

```diff
@@ -88,6 +88,57 @@ Registering the external services with Consul is a multi-step process:
 
 ### Register external services with Consul
 
+There are two ways to register an external service with Consul:
+1. If you have [`TransparentProxy`](/docs/k8s/helm#v-connectinject-transparentproxy) enabled, you can declare external endpoints with the [Destination](/docs/connect/config-entries/service-defaults#terminating-gateway-destination) field of `service-defaults`.
+1. You can add the service as a Node in the Consul catalog.
+
+#### Register an external service as a Destination
+
+Destinations are valid only for [`TransparentProxy`](/docs/k8s/helm#v-connectinject-transparentproxy) mode and allow clients to dial the external service directly.
+Also note these additional limitations when using Destinations to route traffic through a terminating gateway:
+
+| Layer 4 External Services (TCP) | Client dials hostname | Client dials IP |
+|---|---|---|
+| **with TLS** | ✅ Clients can dial successfully. `CAFiles` are not allowed because traffic is already end-to-end encrypted by the client. | ✅ Clients can dial successfully. CAFiles are not allowed because traffic is already end-to-end encrypted by the client. |
+| **no TLS** | ❌ Because the Sidecar is not protocol aware, it will not be able to identify traffic going to the external service. | ✅ Clients can dial successfully. |
+
+| Layer 7 External Services (HTTP, HTTP2 or gRPC) | Client dials hostname | Client dials IP |
+|---|---|---|
+| **with TLS** | ❌ Because traffic is already encrypted before the sidecar, it cannot route as L7 Traffic. | ❌ Because traffic is already encrypted before the sidecar, it cannot route as L7 Traffic. |
+| **no TLS** | ✅ Clients can dial successfully. A `Host` or `:authority` header is required. | ✅ Clients can dial successfully. |
+
+-> **NOTE**: Traffic from unencrypted clients can still be secured from the terminating gateway to the external service by providing a `caFile`.
+See the discussion below under [Create the configuration entry for the terminating gateway](/docs/k8s/connect/terminating-gateways#create-the-configuration-entry-for-the-terminating-gateway).
```

Comment:
```suggestion
You provide a `caFile` to secure traffic between unencrypted clients that connect to external services through the terminating gateway.
Refer to [Create the configuration entry for the terminating gateway](/docs/k8s/connect/terminating-gateways#create-the-configuration-entry-for-the-terminating-gateway) for details.
```
This didn't seem like orthogonal information that should be labeled in a side note. 
Also restated in active voice.

Reply:
I used the note more to emphasize "pay attention" rather than "something related", but happy to adopt the convention here.